fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/event/BiomeLoadingCallback.java

```diff
@@ -0,0 +1,21 @@
+package net.fabricmc.fabric.api.biomes.v1.event;
+
+import net.minecraft.util.registry.RegistryKey;
+import net.minecraft.world.biome.Biome;
+
+import net.fabricmc.fabric.api.biomes.v1.FabricBiomeBuilder;
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public interface BiomeLoadingCallback {
+	Event<BiomeLoadingCallback> EVENT = EventFactory.createArrayBacked(
+			BiomeLoadingCallback.class,
+			(listeners) -> (biomeRegistryKey, biomeBuilder) -> {
+				for (BiomeLoadingCallback callback : listeners) {
+					callback.onBiomeLoading(biomeRegistryKey, biomeBuilder);
+				}
+			}
+	);
+
+	void onBiomeLoading(RegistryKey<Biome> biomeRegistryKey, FabricBiomeBuilder biomeBuilder);
```

Comment:
This would definitely let you add things to biomes. My concern is mod devs use this to change attributes about a biome.

Reply:
I mean, if mod devs want to change attributes, they might as well?

Reply:
I could reduce the scope of the builder to only features, but I don't see how that would make it better.
This builder can also be used to clone existing biomes for re-registering them as something else, in that case you'd be happy to be able to change those attributes.

Reply:
I had one solution around this, split the builder in two and the full builders implements the structure et al stuff from scoped builder

Reply:
I wonder though: Why is changing structures/features different from changing other attributes?
If this is strictly for adding features and structures, we can probably reduce the surface of this API.